lib/services/devices/deviceService.js

```diff
@@ -99,25 +100,34 @@ function createInitialEntityHandler(deviceData, newDevice, callback) {
  */
 function createInitialEntity(deviceData, newDevice, callback) {
     var options = {
-        url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
-            '/v1/updateContext',
-        method: 'POST',
-        json: {
-            contextElements: [
-                {
-                    type: deviceData.type,
-                    isPattern: 'false',
-                    id: String(deviceData.name),
-                    attributes: []
-                }
-            ],
-            updateAction: 'APPEND'
+            url: 'http://' + config.getConfig().contextBroker.host + ':' + config.getConfig().contextBroker.port +
+                '/v1/updateContext',
+            method: 'POST',
+            json: {
+                contextElements: [
+                    {
+                        type: deviceData.type,
+                        isPattern: 'false',
+                        id: String(deviceData.name),
+                        attributes: []
+                    }
+                ],
+                updateAction: 'APPEND'
+            },
+            headers: {
+                'fiware-service': deviceData.service,
+                'fiware-servicepath': deviceData.subservice
+            }
```

Comment:
I saw that in the initial entity creation request no `Fiware-Correlator` is included. Should it be included? Is it pending for a future PR? :) Thanks!